packages/core/src/common/quick-pick-service.ts

```diff
@@ -15,6 +15,10 @@
  ********************************************************************************/
 import { QuickOpenHideReason } from './quick-open-service';
```

Comment:
make sure that this file imports only from `common`, not from `browser`

We wanted to move types in order to fix particularly this place.

Reply:
done